content/md/products-super-power/enrich-your-reference-entity-records.md

```diff
@@ -111,6 +111,35 @@ In fact, the Rich text editor offers a large bunch of possibilities!
 
 ![Records text area](../img/Records_text_area.png)
 
+# Display the products linked to a record (_since 3.2_)
+_Since the 3.2 version_, you can view directly from a reference entity record the first products linked to a record in a new `Products` tab.
+
+To display the 1st products linked to a `Kartell` record:
+1. Go to the `Kartell` record
+1. Then click on the `Products` tab
+1. To open a product, just click on it and it will open the product form in a new tab.
+1. If several product attribute are linked to the `Brand` reference entity, you can switch the product attribute in the `Product attribute` list.
```

Comment:
A mettre plutôt en mode "info" 
Ajouter un "s" au 1er "attribute" 
Pour éviter la répétition de "product attribute", tu peux mettre "you can select the good one in the "product attribute" filter." 
Et donc remplacer "list" par "filter"